x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/CaseTests.java

```diff
@@ -60,6 +60,7 @@ public class CaseTests extends AbstractScalarFunctionTestCase {
         ).collect(Collectors.toList());
         if (Build.current().isSnapshot()) {
             t.add(DataType.DATE_NANOS);
+            t.add(DataType.SEMANTIC_TEXT);
```

Comment:
This should probably go back to adding all "under construction" types.  That was the behavior before https://github.com/elastic/elasticsearch/pull/113920/files#diff-a76076e67449e24e27b719187181b520990bd656fab824d3f0706dbc785f072b and I'd prefer it to having an explicit list here.